src/cli/cli.cpp

```diff
@@ -2050,12 +2050,12 @@ void Interpreter::ProcessNetif(uint8_t aArgsLength, char *aArgs[])
     OT_UNUSED_VARIABLE(aArgs);
 
     otError      error    = OT_ERROR_NONE;
-    const char * netif    = NULL;
+    const char * netif    = nullptr;
     unsigned int netifidx = 0;
 
     SuccessOrExit(error = otPlatGetNetif(mInstance, &netif, &netifidx));
 
-    mServer->OutputFormat("%s:%u\r\n", netif ? netif : "<NULL>", netifidx);
+    mServer->OutputFormat("%s:%u\r\n", netif ? netif : "<nullptr>", netifidx);
```

Comment:
I don't have a strong preference. Any other opinions?

/cc/ @bukepo

Reply:
maybe we follow the convention to print as `(null)`.

Reply:
Agree that `null` or `NULL` is better than `nullptr` because `nullptr` is too implementation-specific.
However, one thing about changing from `<NULL>` to `(null)` is that it has nothing to do with this PR. 
I would prefer to keep `<NULL>` unchanged in this PR.
Thoughts ? @jwhui @bukepo